site/content/en/docs/tasks/run_rayjobs.md

```diff
@@ -0,0 +1,141 @@
+---
+title: "Run A RayJob"
+date: 2022-05-18
+weight: 6
+description: >
+  Run a Kueue scheduled RayJob.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [KubeRay's](https://ray-project.github.io/kuberay/) [RayJob](https://ray-project.github.io/kuberay/guidance/rayjob/).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+By default the integration for `ray.io/rayjob` is not enabled, check the[Installation](/docs/installation/) for details on [Kueue's manger configuration](/docs/installation/#install-a-custom-configured-released-version) 
+
+Check [administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial cluster setup.
+
+Check [KubeRay](https://ray-project.github.io/kuberay/) documentation for installation and configuration details.
+
+## RayJob definition
+
+Besides the usual RayJob configuration while using Kueue take into consideration the following aspects:
+
+### a. Queue selection
+
+The target [local queue](/docs/concepts/local_queue) should be specified in the `metadata.labels` section of the RayJob configuration.
+
+```yaml
+metadata:
+  labels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.rayClusterSpec`.
+
+```yaml
+    headGroupSpec:
+      template:
+        spec:
+          containers:
+            - resources:
+                requests:
+                  cpu: "1"
+    workerGroupSpecs:
+      - template:
+          spec:
+            containers:
+              - resources:
+                  requests:
+                    cpu: "1"
+```
+
+### c. Limitations
+
+- A Kueue managed RayJob cannot use an existing cluster.
+- The RayJob's cluster should be deleted at the end of its execution, `spec.ShutdownAfterJobFinishes` should be `true`.
+- Because Kueue will reserve the RayJob resources, the cluster's `inTreeAutoscaling` should not be enabled.
+- Because a Kueue workload can have a maximum of 8 PodSets, the maximum number of `workerGroupSpecs` is 7.
+
+## Sample RayJob
+
+```yaml
+apiVersion: ray.io/v1alpha1
+kind: RayJob
```

Comment:
Is this inspired by https://github.com/ray-project/kuberay/blob/82c925bb3fe72e056a21521c3b6fff3b172c2817/ray-operator/config/samples/ray_v1alpha1_rayjob.yaml?

If so, it would be worth mentioning that as a comment.

Reply:
It kind of is, but the config map is added and it's `generateName` based  sou you can crete multiple jobs base in the same yaml.